content/influxdb/v2/process-data/common-tasks/_index.md

```diff
@@ -1,17 +1,18 @@
 ---
```

Comment:
This doc shouldn't be modified in the v2 docs. This sections is specifically focused on Flux tasks (scheduled Flux queries).